src/components/Image/Image.js

```diff
@@ -33,15 +33,27 @@ const LazyLoadedImage = (props) => {
     // the low quality image (props.image.thumbnail) will be hidden
   };
 
+  // This block checks whether the user specified fallback image actually works or not
+  const [isLoading, setIsLoading] = useState(true);
+  const [isValidFallback, setIsValidFallback] = useState(false);
+  useEffect(() => {
+    fetch(props.placeholderImg).then(res => {
+      setIsValidFallback(res.status === 200);
+      setIsLoading(false);
+    }).catch((_reason) => {
```

Comment:
Here, `finally` statement can be used to combine `setIsLoading(false);`.

Reply:
This block has now been moved out of this file, and is used in such a way now that `finally` should no longer be needed.